pages/post.js

```diff
@@ -0,0 +1,15 @@
+import { useRouter } from "next/router";
+import Layout from "../comps/MyLayout";
+
+const Page = () => {
```

Comment:
By convention, this should be `Post` since it's inside a `post.js` file … not a huge deal, it obviously works